docker-compose.yml

```diff
@@ -2,7 +2,7 @@ version: '3'
 
 services:
   web:
-    image: buoyantio/emojivoto-web:v11
+    image: buoyantio/emojivoto-web:v11.1
```

Comment:
This repo was forked from the buoyantio one, but when pushing we're not updating the images on their registry. `buoyantio/emojivoto-web:v11.1` won't resolve to our updated image, it should be `datawire/<service>:v11.1`.